book/src/installers/homebrew.md

```diff
@@ -2,13 +2,34 @@
+cargo-dist can automatically build and publish [Homebrew](https://brew.sh) formulae (packages) for your application. Users can install your application with an expression like `brew install axodotdev/tap/axolotlsay` and automatically get updates whenever they update their Homebrew packages. The homebrew package will fetch your prebuilt [archives](../artifacts/archives.md), and install them in the traditional homebrew directory structure.
 
-cargo-dist can, optionally, publish your formula to a tap repository for you on every release. To enable this, add a `tap` field to your `Cargo.toml` pointing to a GitHub repository that you control and add `homebrew` to the `publish-jobs` field. The repository name must start with `homebrew-`. For example:
+*Building* a formula is pretty straight-forward, but publishing it requires you to create a your own [Homebrew tap](https://docs.brew.sh/Taps) (package repository), because [the core homebrew tap](https://github.com/Homebrew/homebrew-core) does not accept prebuilt binaries from third parties. Thankfully, making your own tap is as simple as creating a repository named "homebrew-tap" under your GitHub org or user, and getting a GitHub API token to push to it. cargo-dist will manage the contents of the repo for you.
+
+
+## Quickstart Part 1: Creating A Custom Tap
+
+1. Create a GitHub repository called "homebrew-tap" (`axodotdev/homebrew-tap`)
+2. Create a GitHub [personal access token](https://github.com/settings/tokens/new?scopes=repo) with the `repo` scope
+3. Add the token as a [GitHub Secret](https://docs.github.com/en/actions/security-guides/encrypted-secrets) called `HOMEBREW_TAP_TOKEN` to the repository you want to publish **from** (`axodotdev/axolotlsay`)
+
+We recommend initializing the repository with a README, but otherwise the directory structure will be managed by cargo-dist, and many separate repos can publish to the same tap without issue.
+
+A Homebrew Tap is just a GitHub repository that starts with `homebrew-`. Many homebrew features allow that prefix to be elided, so the package `axolotlsay` published in `axodotdev/homebrew-tap`, can be installed as `axodotdev/tap/axolotlsay`. Your users don't need to "register" anything to use it, custom taps are just that builtin to Homebrew.
+
+
+## Quickstart Part 2: Enabling The Homebrew Installer
+
+1. run `cargo dist init` on your project
+2. when prompted to pick installers, enable "homebrew"
+3. this should trigger a prompt for your tap (`axodotdev/homebrew-tap`)
+
+...that's it! Assuming you already setup your custom tap, as described in the previous section. If this worked, your config should now contain the following entries:
 
 ```toml
 [workspace.metadata.dist]
-tap = "axodotdev/homebrew-formulae"
+installers = [..., "homebrew", ...]
```

Comment:
If we're going to elide parts of the toml such that it's not valid, we should turn off syntax highlighting for the block.